recipes/bcftools-gtc2vcf-plugin/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-export VERSION=1.16
+export VERSION=1.18
```

Comment:
```suggestion
export VERSION=${PKG_VERSION}
```
I think using the `PKG_VERSION` variable is more efficient here; the `VERSION` string will not need editing in future releases